js-miniapp-sdk/CHANGELOG.md

```diff
@@ -2,7 +2,10 @@
 
 ### v1.2.0 (In progress)
 
-- Added support for requesting show interstitial ad from the host application to allow displaying Interstitial ads [See here](README.MD#4-Show-Interstitial-Ad)
+- Added support for requesting show ads from the host application to allow displaying Interstitial and Rewarded ads [See here](README.MD#4-Show-Ads)
+
+- Added support for requesting load ads from the host application to allow loading Interstitial and Rewarded ads [See here](README.MD#5-Load-Ads)
```

Comment:
Maybe these should be combined into a single line: `Added support for requesting to load and show ads ....`